cmd/iam.go

```diff
@@ -622,8 +622,8 @@ func (sys *IAMSys) Init(ctx context.Context, objAPI ObjectLayer) {
 		}
 
 		// Migrate IAM configuration, if necessary.
-		if err := sys.doIAMConfigMigration(ctx); err != nil {
-			txnLk.Unlock(cancel)
+		if err := sys.doIAMConfigMigration(lkctx.Context()); err != nil {
```

Comment:
Frankly I did not want for this to have a context because this is a migration, context.Background() could be better

Reply:
Sure